.github/workflows/linux-check.yaml

```diff
@@ -163,5 +163,14 @@ jobs:
           # world_feed_integration_tests - https://github.com/organicmaps/organicmaps/issues/215
           CTEST_EXCLUDE_REGEX: "drape_tests|generator_integration_tests|opening_hours_integration_tests|opening_hours_supported_features_tests|routing_benchmarks|routing_integration_tests|routing_quality_tests|search_quality_tests|storage_integration_tests|shaders_tests|world_feed_integration_tests"
         run: |
+          sudo locale-gen en_US
+          sudo locale-gen en_US.UTF-8
+          sudo locale-gen es_ES
+          sudo locale-gen es_ES.UTF-8
+          sudo locale-gen fr_FR
+          sudo locale-gen fr_FR.UTF-8
+          sudo locale-gen ru_RU
+          sudo locale-gen ru_RU.UTF-8
+          sudo update-locale
```

Comment:
I think it's better to move this part to a separate job. Probably near the dependencies job or before the tests job with a comment that it's required only for testing

Reply:
Hi @AndrewShkrob! I've removed the UTs for the `ToStringPrecision()` function, so these calls to `sudo locale-gen XX_YY` are no longer needed. Thanks anyway for your comment.

Reply:
Hello again @AndrewShkrob! As I've included back again the UTs for the  `ToStringPrecision()` function, I've included the `sudo locale-gen xx_YY` calls in the Github workflow for Linux, in the same place that I initially used.

You suggested to move this part to a separate job, but I'm not sure how to do it.

Currently, Linux tests are run in the last step 'Tests' of the 'linux-matrix' job.

I'd appreciate if you could provide a piece of code of your proposal for this change.

Reply:
1. You can move these calls to the tests section. 
2. Is there a way to detect if locale is installed and skip the test if it's not?
3. Is there a way to get a locale-specific delimiter from a test and avoid ifdef?

Reply:
1. I still don't know where you want these calls to be placed. They already are located in the 'Tests' step of the 'linux-matrix' job, and I don't see any other tests section in this 'linux-check.yaml" file. Please provide a diff file or some direct instructions (like: insert these calls in line XX of file YY), otherwise I'm lost regarding this change request.

2. I've added some try/catch in the `GetLocale()` function in Linux to detect errors while loading the locales, so now the tests don't crash and we can skip the tests for the missing locales. Under macOS, `GetLocale()` never fails, even if a wrong locale is requested.

3. I've removed the `#ifdefs` for grouping separator checking in different OS's by adding a `AddGroupingSeparators()` function that uses locale's grouping separator for the tests.